InternalTimeSheet/app/src/main/java/br/com/accera/internaltimesheet/ui/main/MainActivity.java

```diff
@@ -59,7 +94,63 @@ protected void onDataBindingReady(ViewDataBinding coreDataBinding) {
 
         PushDownAnim.setPushDownAnimTo(binding.start)
                 .setOnClickListener(view ->
-                        mCorePresenter.receiveClick(binding.getCadastro().Nome, binding.getCadastro().Diainit, binding.getCadastro().Intervaloinit, binding.getCadastro().Intervaloend, binding.getCadastro().Diaend));
+                        mCorePresenter.receiveClick(binding.getCadastro().Nome, binding.getCadastro().Diainit, binding.getCadastro().Intervaloinit, binding.getCadastro().Intervaloend, binding.getCadastro().Diaend)
```

Comment:
You could create a local variable to "getCadastro" method. When you call this method many times you are forcing the processor work